pyproject.toml

```diff
@@ -149,7 +149,8 @@ exclude = [
     "lib/",
     "etl/steps/archive",
     "etl/snapshots/archive",
-    "apps/wizard/templating/cookiecutter",
+    "apps/wizard/etl_steps/cookiecutter/",
```

Comment:
Let's remove pywebio from pyproject.py.

Reply:
what do you think about leaving the old fast-track here for a couple of weeks to see if anything goes wrong with the new version?